paasta_tools/kubernetes_tools.py

```diff
@@ -2405,6 +2413,29 @@ def list_deployments(
     ]
 
 
+def list_deployments_in_managed_namespaces(
+    kube_client: KubeClient,
+    label_selector: str,
+) -> List[KubeDeployment]:
+    ret: List[KubeDeployment] = []
+    for namespace in get_all_managed_namespaces(kube_client):
+        try:
+            ret.extend(
+                list_deployments(
+                    kube_client=kube_client,
+                    label_selector=label_selector,
+                    namespace=namespace,
+                )
+            )
+        except ApiException as exc:
+            log.error(
+                f"Error fetching deployments from namespace {namespace}: "
+                f"status: {exc.status}, reason: {exc.reason}."
+            )
```

Comment:
I assume we're not using log.exception here as we're generally not actually too interested in the full traceback here?